content/locale/index.js

```diff
@@ -47,3 +47,37 @@ function getLocale(lang) {
 			return 'en'
 	}
 }
+
+const isPlainObject = obj => {
+	return Object.prototype.toString.call(obj) === '[object Object]'
+}
+
+export function fallback() {
```

Comment:
en.js
```
{
    a-key: 'Message for A'
    b-key: 'Message for B'
}
```
fr.js
```
{
    a-key: 'Translation for A'
}
```
this is handling fallback for a case like this right?

Reply:
yes right!

Reply:
just the en.js's b-key will add to fr.js.

Reply:
> btw is there any logs or information when the language (fr.js) does not provide the translation?
Hi ahnafalfariza,
just the b-key and the value of en.js will add to fr.js.